libs/dh/shared/environments/src/lib/app-environment/dh-app-environment.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { InjectionToken } from '@angular/core';
+import { dhLocalAppEnvironment } from '@energinet-datahub/dh/shared/assets';
+
+import { environment } from '../environment';
+
+export interface DhAppEnvironmentConfig {
+  current: string;
+}
+
+export enum DhAppEnvironments {
+  local = 'localhost',
+  dev = 'u-001',
+  test = 't-001',
+  preProd = 'b-001',
+  prod = 'b-002',
+}
+
+export const dhAppEnvironmentToken = new InjectionToken<DhAppEnvironmentConfig>(
+  'dhAppEnvironmentToken',
+  {
+    factory: (): DhAppEnvironmentConfig => {
+      if (environment.production) {
+        throw new Error('No DataHub app environment config provided.');
+      }
+
+      // Used for unit and integration tests
+      return dhLocalAppEnvironment as unknown as DhAppEnvironmentConfig;
```

Comment:
Reason is because we're loading the config from json, typescript sees it as a `string` which is not `DhAppEnvironment`. If you remove it you should see an error.

Reply:
`resolveJsonModule` is actually already set to true in `tsconfig.base.json`, so properly it's something else which causes the issue. If you wanna give it a try @dzhavat or @ManBearTM please do :)

Reply:
Ah, it's type incompatibility. `DhAppEnvironmentConfig.current` is `DhAppEnvironment` whereas the `dhLocalAppEnvironment.current` is seen as `string` by TS.